tools/ClassGen/NodeGen.cpp

```diff
@@ -311,6 +311,9 @@ int main(int argc, char **argv) {
       .setDocstring("Performs element-wise natural log to the Input.");
   // clang-format on
 
+  BB.newNode("Exp").addInput("Input").addResultFromCtorArg().setDocstring(
```

Comment:
Can you  disable clang-format for this Node since it gets formatted all weird? See just above for "Log".

Reply:
will do that.